src/modules/navigator/navigator_main.cpp

```diff
@@ -944,6 +953,20 @@ Navigator::get_acceptance_radius(float mission_item_radius)
 	return radius;
 }
 
+float
+Navigator::get_yaw_acceptance(float mission_item_yaw)
+{
+	float yaw = mission_item_yaw;
+
+	const position_controller_status_s &pos_ctrl_status = _position_controller_status_sub.get();
+
+	if ((pos_ctrl_status.timestamp > _pos_sp_triplet.timestamp) && !PX4_ISFINITE(pos_ctrl_status.yaw_acceptance)) {
```

Comment:
Is the logic wrong here? `!PX4_ISFINITE`

Reply:
If I set the `yaw_acceptance` to NaN in the position controller, I want to overwrite it so that navigator always sets `_waypoint_yaw_reached` to true. In this way a waypoint can be reached from any direction. 

Do you think there are use cases where it's necessary to increase the `yaw_acceptance` angle without setting it to NaN?

Reply:
Oh okay, I thought the usage was opposite. If pos_ctrl_status.yaw_acceptance is not NAN (ie valid) then use it.
Could you add a comment explaining the intent? Explicitly setting `yaw = NAN` might also help make it clear.

Another idea would be to make it a flag for what you actually want. Yaw acceptance ignored/disabled/not possible/etc.